### PR TITLE
add license copyright update

### PR DESCRIPTION
**Pull request by alexpantyukhin**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by alexpantyukhin** [comment]
https://github.com/TheAlgorithms/scripts/issues/21

**Review by Panquesito7 — APPROVED** [review]
Looking great. Thanks! 🚀
Although, wouldn't it be better to add it to the [scripts](https://github.com/TheAlgorithms/scripts) repository? 🤔

**Comment by alexpantyukhin** [comment]
This is good idea!

**Comment by alexpantyukhin** [comment]
https://github.com/TheAlgorithms/scripts/pull/22

File: .github/workflows/license.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  schedule:
+    - cron: '0 0 1 1 *'
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update the License
+        run: |
+          (echo "Copyright (C) 2016-$(date +"%Y") TheAlgorithms and contributors";  tail -n +2 License) > License.tmp
+          mv License.tmp License
```
Comment (tjgurwara99) [inline .github/workflows/license.yml:12]:
```suggestion
          (echo "Copyright (C) 2016-$(date +"%Y") TheAlgorithms and contributors";  tail -n +2 LICENSE) > License.tmp
          mv License.tmp LICENSE
```

**Review by Panquesito7 — APPROVED** [review]

**Comment by tjgurwara99** [comment]
I think we should wait till the reusable workflow is done in the scripts repo and use that one here instead of rewriting it here. I'm closing this PR @alexpantyukhin, please could you open a separate PR once the reusable workflow is patched with the suggestions mentioned in that PR? Then use it the way I have created the test workflow in the patch file 😄

**Pull request closed without merging**